core/framework/volume.go

```diff
@@ -0,0 +1,26 @@
+// Copyright 2021 Red Hat, Inc. and/or its affiliates
```

Comment:
maybe rename to volume_utils?

Reply:
The entire `framework` package is a utility package for interactions with the underlying API.